internal/events/batch_pin_complete.go

```diff
@@ -33,28 +33,28 @@ import (
 //
 // We must block here long enough to get the payload from the publicstorage, persist the messages in the correct
 // sequence, and also persist all the data.
-func (em *eventManager) BatchPinComplete(bi blockchain.Plugin, batchPin *blockchain.BatchPin, signingIdentity string, protocolTxID string, additionalInfo fftypes.JSONObject) error {
+func (em *eventManager) BatchPinComplete(bi blockchain.Plugin, batchPin *blockchain.BatchPin, author string, protocolTxID string, additionalInfo fftypes.JSONObject) error {
```

Comment:
For consistency in terminology, isn't this what we're calling a "key" (ie an Ethereum address)? Seems like the word "author" has evolved to mean a higher level FireFly/DID string.